pcawg_metadata_parser/release_oct2015/excluded_gnos_ids.txt

```diff
@@ -0,0 +1,5 @@
+3c78eef3-d385-4c9f-81cd-d2948ef00f05
+ffe3eee8-2b5e-4eb2-b42e-7d9d2e4b2203
+a19ba8e0-8ca0-40b8-b40e-92446d8d347b
+f3542b74-965a-48d7-a707-3b0a04f173ac
+c4cd1ffc-6f9a-4c6f-98ad-55997a73245d
```

Comment:
These should be the excluded gnos ids when we generated oct2015 release. They may not be effective in future as we have all the gnos_ids in oct2015 release.

-Linda




On Nov 20, 2015, at 3:50 PM, Junjun Zhang <notifications@github.com<mailto:notifications@github.com>> wrote:


In pcawg_metadata_parser/release_oct2015/excluded_gnos_ids.txt<https://github.com/ICGC-TCGA-PanCancer/pcawg-central-index/pull/43#discussion_r45516819>:

> @@ -0,0 +1,5 @@
> +3c78eef3-d385-4c9f-81cd-d2948ef00f05
> +ffe3eee8-2b5e-4eb2-b42e-7d9d2e4b2203
> +a19ba8e0-8ca0-40b8-b40e-92446d8d347b
> +f3542b74-965a-48d7-a707-3b0a04f173ac
> +c4cd1ffc-6f9a-4c6f-98ad-55997a73245d


what is this file: pcawg_metadata_parser/release_oct2015/excluded_gnos_ids.txt?

—
Reply to this email directly or view it on GitHub<https://github.com/ICGC-TCGA-PanCancer/pcawg-central-index/pull/43/files#r45516819>.